inference/core/workflows/core_steps/visualizations/classification_label/v1.py

```diff
@@ -0,0 +1,525 @@
+
+3. 🎨 **Rendering** labels with customizable positioning and styling
+#### Parameters
+
+- **`task_type`**: Specifies how to handle predictions. Available options:
+  
+    * **"single-label"**: Shows only the highest confidence prediction
+  
+    * **"multi-label"**: Displays all predictions above threshold
+
+- **`text_position`**: Controls label placement with 9 options:
+  
+    * **Top**: LEFT, CENTER, RIGHT
+    * **Center**: LEFT, CENTER, RIGHT
+    * **Bottom**: LEFT, CENTER, RIGHT
+
+- **`text`**: Determines what information to display:
+  
+    * **"Class"**: Only show class names
+    * **"Confidence"**: Only show confidence scores
+    * **"Class and Confidence"**: Show both
+
+- **`text_padding`**: Controls spacing between labels and from image edges
+
+#### Why Use This Visualization?
+
+This is especially useful for:
+
+- 🏷️ Creating clear, professional-looking prediction displays
+
+- 📱 Supporting different UI layouts with flexible positioning
+
+- 🎨 Customizing appearance for different use cases
+
+- 📊 Showing prediction confidence in an intuitive way
+
+#### Example Usage
+
+Use this visualization after any classification model to display predictions in a clean, 
+organized format. Perfect for both single predictions and multiple class probabilities.
+"""
+
+class ClassificationLabelManifest(ColorableVisualizationManifest):
+    type: Literal["roboflow_core/classification_label_visualization@v1"]
+    model_config = ConfigDict(
+        json_schema_extra={
+            "name": "Classification Label Visualization",
+            "version": "v1",
+            "short_description": SHORT_DESCRIPTION,
+            "long_description": LONG_DESCRIPTION,
+            "license": "Apache-2.0",
+            "block_type": "visualization",
+        }
+    )
+
+    text: Union[
+        Literal[
+            "Class",
+            "Confidence",
+            "Class and Confidence"
+        ],
+        WorkflowParameterSelector(kind=[STRING_KIND]),
+    ] = Field(  # type: ignore
+        default="Class",
+        description="The type of text to display.",
+        examples=["LABEL", "$inputs.text"],
+    )
+
+    text_position: Union[
+        Literal[
+            "CENTER",
+            "CENTER_LEFT",
+            "CENTER_RIGHT",
+            "TOP_CENTER",
+            "TOP_LEFT",
+            "TOP_RIGHT",
+            "BOTTOM_LEFT",
+            "BOTTOM_CENTER",
+            "BOTTOM_RIGHT",
+        ],
+        WorkflowParameterSelector(kind=[STRING_KIND]),
+    ] = Field(  # type: ignore
+        default="TOP_LEFT",
+        description="The anchor position for placing the label.",
+        examples=["CENTER", "$inputs.text_position"],
+    )
+
+    predictions: StepOutputSelector(kind=[CLASSIFICATION_PREDICTION_KIND]) = Field(  # type: ignore
+        description="Classification predictions.",
+        examples=["$steps.classification_model.predictions"],
+    )
+
+    task_type: Literal["single-label", "multi-label"] = Field(
```

Comment:
examples needed for all fields

+ please provide JSON with description of task types

Reply:
not sure if that's even needed, should be deduced from content of predictions - they have visibly different payloads

Reply:
I removed task_type. You're right, its redundant.